simulator/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM rust:1.70.0-slim-bullseye as builder
+
+RUN apt update && apt install -y \
+    cmake=3.18.4-2+deb11u1 \
+    clang=1:11.0-51+nmu5 \
+    build-essential=12.9 \
+    llvm=1:11.0-51+nmu5
+#RUN rustup target add wasm32-unknown-unknown
+
+ADD simulator /app/simulator
+ADD radix-engine /app/radix-engine
+ADD radix-engine-interface /app/radix-engine-interface
+ADD sbor /app/sbor
+ADD sbor-derive /app/sbor-derive
+ADD sbor-derive-common /app/sbor-derive-common
+ADD utils app/utils
+ADD scrypto-schema /app/scrypto-schema
+ADD radix-engine-common /app/radix-engine-common
+ADD radix-engine-derive /app/radix-engine-derive
+ADD native-sdk /app/native-sdk
+ADD radix-engine-macros /app/radix-engine-macros
+ADD radix-engine-profiling /app/radix-engine-profiling
+ADD radix-engine-store-interface /app/radix-engine-store-interface
+ADD radix-engine-stores /app/radix-engine-stores
+ADD transaction /app/transaction
+ADD radix-engine-queries /app/radix-engine-queries
+ADD assets /app/assets
```

Comment:
I think we have a recursively dependency here? The assets will be built by the docker image and the docker image depends on the "previous" assets. 

As long as the final WASM files doesn't depend on the assets, we're good. I think this is the case, but maybe worth double checking?

Reply:
I'll check but I think there's a dependency between simulator and assets. without assets simulator was failing to build. will check again

Reply:
@iamyulong so the problem when removing assets is that I can't compile the radix-engine. 
Getting this error
```
62.41 error: could not compile `radix-engine` (lib) due to 5 previous errors
62.41 warning: build failed, waiting for other jobs to finish...
129.5 error: failed to compile `simulator v0.13.0 (/app/simulator)`, intermediate artifacts can be found at `/app/simulator/target`
```
is that expected?

Reply:
Let's ignore this for now. The simulator does depend on the assets.